src/components/ProjectsTable/ProjectTable.tsx

```diff
@@ -0,0 +1,162 @@
+  const [deleteProjectMutation] = useMutation<{ affected: number }>(
+    DELETE_PROJECT
+  );
+
+  const [sorting, setSorting] = useState<SortingType>({
+    name: 'internal_name',
+    asc: true
+  });
+
+  const [anchorEl, setAnchorEl] = useState<AnchorType>(initialAnchor);
+
+  const handleMenuClose = () => {
+    setAnchorEl(initialAnchor);
+  };
+
+  const handleMenuOpen = (
+    event: React.MouseEvent<SVGSVGElement>,
+    Id: string
+  ) => {
+    setAnchorEl({ anchor: event.currentTarget, Id });
+  };
+
+  const changeSort = (label: LabelsType) => {
+    setSorting((prev) => ({
+      name: label,
+      asc: prev.name === label ? !prev.asc : true
+    }));
+  };
+
+  const deleteProject = async () => {
+    await deleteProjectMutation({
+      variables: { id: anchorEl.Id }
+    });
+  };
+
+  if (loading) {
+    return <Loader />;
+  }
+
+  if (error) {
+    return <div>{error.message}</div>;
+  }
+
+  return (
+    <>
+      <Menu
+        anchorEl={anchorEl.anchor}
+        open={!!anchorEl.anchor}
+        onClose={handleMenuClose}
+      >
+        <MenuItem>Project</MenuItem>
+        <MenuItem onClick={deleteProject} disabled={!isUserAdmin}>
+          Delete project
+        </MenuItem>
+      </Menu>
+      <TableContainer component={Paper}>
+        <Table>
+          <TableHead>
+            <TableRow>
+              {headerLabels.map((label) => (
+                <StyledTableCell
+                  key={label.label}
+                  onClick={() => changeSort(label.value)}
+                >
+                  {label.label}
+                  {sorting.name === label.value &&
+                    (sorting.asc ? (
+                      <ArrowUpwardIcon fontSize="small" />
+                    ) : (
+                      <ArrowDownwardIcon fontSize="small" />
+                    ))}
+                </StyledTableCell>
+              ))}
+              <StyledTableCell padding="checkbox" />
+            </TableRow>
+          </TableHead>
+          <StyledTableBody>
+            {sortUsers(filterUsers(data.projects, search), sorting).map(
+              (project) => (
+                <TableRow key={project.id}>
+                  <TableCell>{project.name}</TableCell>
+                  <TableCell>{project.internal_name}</TableCell>
+                  <TableCell>{project.domain}</TableCell>
+                  <TableCell>{project.start_date}</TableCell>
+                  <TableCell>{project.end_date || 'Till now'}</TableCell>
+                  <TableCell>{project.team_size}</TableCell>
+                  <TableCell>
+                    <IconButton>
+                      <MoreVertIcon
+                        onClick={(event) => handleMenuOpen(event, project.id)}
+                      />
+                    </IconButton>
+                  </TableCell>
+                </TableRow>
+              )
+            )}
+          </StyledTableBody>
+        </Table>
+      </TableContainer>
+    </>
+  );
+};
+
```

Comment:
looks good, but soon it would be better to create a reusable table (usersTable looks the same and there will be anothers tables in the app)